etc/config/cobol.amazon.properties

```diff
@@ -1,15 +1,21 @@
 compilers=&gnucobol
-defaultCompiler=gnucobol32rc2
+defaultCompiler=gnucobol31
 
 group.gnucobol.groupName=GnuCOBOL
-group.gnucobol.compilers=gnucobol32rc2:gnucobol31rc1
+group.gnucobol.compilers=gnucobol32rc2:gnucobol31:gnucobol22:gnucobol11
```

Comment:
In general we tend to never delete old compilers, as that invalidates saved URLs, but as it's likely only you using this we can do it. I'll build and install those compilers now, and check they installed ok before I merge this. Thanks!